src/app/service/todo-http.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ToDoItem } from 'src/model/ToDoItem';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class TodoHttpService {
+  constructor(private httpClient: HttpClient) {}
+
+  todoItemURL = 'https://localhost:44309/ToDoItem'
```

Comment:
Suggestion: change ToDoItem into ToDoItems and modify the backend to make API RESTful